src/Liquid.OnAzure/MessageBuses/ServiceBus.cs

```
@@ -63,6 +63,30 @@ public Task ExceptionReceivedHandler(ExceptionReceivedEventArgs exceptionReceive
             ((LightTelemetry)WorkBench.Telemetry).TrackException(exceptionReceivedEventArgs.Exception);
             return Task.CompletedTask;
         }
+		
+        /// <summary>
+        /// Create queue if not exists
+        /// </summary>
+        /// <param name="queue"></param>
+        /// <returns></returns>
+        private async Task<QueueClient> GetOrCreateQueueAsync(KeyValuePair<MethodInfo, QueueAttribute> queue)
+        {
+            ReceiveMode receiveMode = (queue.Value.DeleteAfterRead) ? ReceiveMode.ReceiveAndDelete : ReceiveMode.PeekLock;
+
+            ManagementClient client = new ManagementClient(GetConnection(queue));
+
+            if (await client.TopicExistsAsync(queue.Value.QueueName))
```

There's a race condition here, since we can expect multiple pods to be deployed.... we must ignore errors in case the queue / topic exists once the creation method is called.